perceval/utils/statevector.py

```diff
@@ -96,8 +96,6 @@ def _partition(one_list: list, distribution: list, current: list, all_res: list)
 BasicState = FockState
 BasicState.__add__ = _fockstate_add
 BasicState.__sub__ = _fockstate_sub
-BasicState.__pow__ = _fockstate_pow  # Using issue #210 fix before moving the fix to exqalibur
-BasicState.partition = _fockstate_partition  # TODO use the cpp version of this call
```

Comment:
Could you also remove the code of `_fockstate_pow` and `_fockstate_partition` free functions?